client/src/components/Screens/Login/Login.jsx

```diff
@@ -23,16 +23,22 @@ const Login = () => {
         email,
         password,
       });
+      console.log(response);
 
-      const { userToken } = response.data; // Extract user token from response
+      const { userToken, userId } = response.data; // Extract user token and userId from response
       console.log("Token received:", userToken);
```

Comment:
same here and all the console.log statements

Reply:
okay